sky/server/constants.py

```diff
@@ -17,6 +17,8 @@
 DEFAULT_HANDLER_NAME = 'default'
 # The path to the API request database.
 API_SERVER_REQUEST_DB_PATH = '~/.sky/api_server/requests.db'
+# The path to the cached DAG database.
+API_SERVER_DAG_DB_PATH = '~/.sky/api_server/dags.db'
```

Comment:
how about we keep everything in the same db file to make it easier to migrate in the future? we can have diffrent tables.